SitemapElementInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle;
+
+/**
+ * Documents persisted with phpcr-odm should implement that interface.
+ * They also need to create a property with the name "visible_for_sitemap" to be recognized
+ * by the default provider, that generates the UrlInformation object.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
+ */
+interface SitemapElementInterface
```

Comment:
one potential problem i see with this: there could be more than one sitemap, e.g. an overview for human visitors and the xml sitemap to help google. visibility could depend on the type of sitemap...

i don't want to solve this problem in this PR, but maybe we can quickly think about it to avoid having a solution that blocks further flexibility.